src/Microsoft.DocAsCode.HtmlToPdf/HtmlToPdfConverter.cs

```diff
@@ -195,16 +197,23 @@ private void CreateOutlines(Dictionary<string, object> rootOutline, IList<HtmlMo
 
                                 if (!pdfModel.PageNumber.HasValue)
                                 {
-                                    pdfModel.PageNumber = _currentNumberOfPages - 1;
```

Comment:
Have some discussion with @partychen , could this fix work with a simple change to remove `-1` here?

```
pdfModel.PageNumber = _currentNumberOfPages
```

Reply:
Done. Thank you.